### PR TITLE
Enable configuration of the recording settings

### PR DESCRIPTION
**Pull request by louim**
This is a follow up to https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/81. I followed @BrutalBirdie suggestion of pulling and merging the configuration before updating it, allowing a greater flexibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | `bbb_mute_on_start:` | start with muted mic on join | `no` | |
 | `bbb_app_log_level:` | set bigbluebutton log level | `DEBUG` | |
 | `bbb_meteor:` | overwrite settings in meteor | `{}` | |
+| `bbb_recording_config:` | overwrite recording settings | `{}` | See [Enable playback of recordings on iOS](https://docs.bigbluebutton.org/admin/customize.html#enable-playback-of-recordings-on-ios). It works like `bbb_meteor` by merging your custom config with the server config. |
```
Comment (EdJoPaTo) [inline README.md:62]: should the value name contain the colon? Looks like other values also include it.

Comment (louim) [reply README.md:62]: Hey @EdJoPaTo, do you mean adding it to `bbb_meteor` in the comment column?

Comment (EdJoPaTo) [reply README.md:62]:
```suggestion
| `bbb_recording_config` | overwrite recording settings | `{}` | See [Enable playback of recordings on iOS](https://docs.bigbluebutton.org/admin/customize.html#enable-playback-of-recordings-on-ios). It works like `bbb_meteor` by merging your custom config with the server config. |
```

A bunch of names end with the colon. Not only the one you added here. I wondered If that should be removed. But as its not only in your line that would be a change independent from this PR.

Comment (louim) [reply README.md]: Gotcha. I'll do a separate PR to fix the other rows.

File: tasks/config.yml [hunk 2]
```diff
@@ -93,6 +93,25 @@
...
+- name: combine recording config
+  set_fact:
+    recording_config: "{{ recording_config | combine(bbb_recording_config, recursive=true) }}"
+
+- name: write back recording config
+  copy:
+    content: '{{ recording_config | to_nice_yaml(indent=2) }}'
+    dest: /usr/local/bigbluebutton/core/scripts/presentation.yml
+    mode: '0644'
+
```
Comment (crosscodr) [inline tasks/config.yml:114]:
```suggestion
- name: configure custom recording config
  block:
    - name: read recording config
      slurp:
        path: /usr/local/bigbluebutton/core/scripts/presentation.yml
      register: r_recording_config
    
    - name: extract recording config
      set_fact:
        recording_config: "{{ r_recording_config['content'] | b64decode | from_yaml }}"
    
    - name: combine recording config
      set_fact:
        recording_config: "{{ recording_config | combine(bbb_recording_config, recursive=true) }}"
    
    - name: write back recording config
      copy:
        content: '{{ recording_config | to_nice_yaml(indent=2) }}'
        dest: /usr/local/bigbluebutton/core/scripts/presentation.yml
        mode: '0644'
  when: bbb_recording_config | lenght > 0
```

**Review by crosscodr — CHANGES_REQUESTED** [review]

**Comment by crosscodr** [comment]
Hi @louim, thank you for your PR!
I have a suggestion, to only run these tasks if a custom recording config is defined.

**Review by crosscodr — APPROVED** [review]

**Review by Davka — APPROVED** [review]

**Pull request merged**